src/Symfony/Component/Cache/Tests/Adapter/PredisAdapterTest.php

```diff
@@ -24,7 +24,7 @@ public static function setUpBeforeClass(): void
 
     public function testCreateConnection()
     {
-        $redisHost = getenv('REDIS_HOST');
+        $redisHost = getenv('REDIS_HOST') ?: 'localhost';
```

Comment:
this should not be required, because the env var is found in the phpunit.xml.dist file

Reply:
No, It's not required. But If REDIS_HOST is set in the computer system or in phpunit.xml.dist. The test is failed.

Reply:
Please try these instructions below:

```bash
export REDIS_HOST=another_than_localhost
./phpunit src/Symfony/Component/Cache/Tests
```

Reply:
I updated the PR description. Please see it for more details.